packages/features/sales-escrows/modals/ConfirmSalesActionModal.tsx

```diff
@@ -193,17 +186,19 @@ export const ConfirmSalesActionModal = ({
                 horizontal: 'right',
               },
             });
-            setIsLoading(false);
-            return handleClose(true);
           }
-          setIsLoading(false);
-          return handleClose(false);
         } catch (e) {
-          console.log(e);
+          debug.log(e);
         }
       }
+      handleClose(false);
+    } catch (e) {
+      debug.log(e);
+    } finally {
+      setInProcess?.(false);
+      setIsLoading(false);
+      handleClose(false);
```

Comment:
Please remove the false parameter. Nothing that consumes this components uses it.